### PR TITLE
Add docformatter and pylint to python tooling

### PR DESCRIPTION
**Pull request by ChristopheDuong**
## What
Adding another formatter for documentation in python
and pylint (more extensive of python best practices)

### CODE REVIEW

**Comment by ChristopheDuong** [comment]
@jrhizor I'm not sure how i can call the pylint command yet

because you are supposed to provide python files on command line or directory with __init__.py files but we don't always have some everywhere...

Would you have an idea?

File: tools/gradle/commons/integrations/python.gradle [hunk 1]
```diff
@@ -27,11 +29,22 @@ task isortFormat(type: PythonTask) {
...
     command = ". --config ${rootProject.file('tools/python/.flake8').absolutePath}"
 }
 
+task pylintCheck(type: PythonTask, dependsOn: blackFormat) {
+    module = "pylint"
+    // FIXME this is not working yet...
+    command = "--rcfile=${rootProject.file('tools/python/pylintrc').absolutePath}"
+}
+
 if(project.ext.has('pyModule')) {
```
Comment (jrhizor) [inline tools/gradle/commons/integrations/python.gradle:48]: I think this is the sort of guard necessary for pylint? The folder/module name passed in with `pyModule` (example: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/base-singer/build.gradle#L1)

File: tools/gradle/commons/integrations/python.gradle [hunk 2]
```diff
@@ -45,3 +58,6 @@ build.dependsOn installReqs
 check.dependsOn blackFormat
 check.dependsOn isortFormat
 check.dependsOn flakeCheck
+
+task pythonFormat(dependsOn: ['blackFormat', 'isortFormat', 'docFormat'])
```
Comment (jrhizor) [inline tools/gradle/commons/integrations/python.gradle:62]: I like this. You can also replace the `check.dependsOn` calls with `check.dependsOn pythonFormat` and `check.dependsOn pythonLint`.

**Comment by cgardens** [comment]
@ChristopheDuong I think this got incorporated into a separate PR. should we close this one?

**Pull request closed without merging**